react/components/BuyButton/index.js

```diff
@@ -45,8 +45,9 @@ export class BuyButton extends Component {
   }
 
   handleAddToCart = async () => {
-    const { skuItems, isOneClickBuy, orderFormContext, push } = this.props
+    const { skuItems, isOneClickBuy, orderFormContext, push, onAddPress, onAddFinish } = this.props
```

Comment:
perhaps switch from `onAddPress` to `onAddStart`, in order to be in line with `onAddFinish`

Reply:
up